AssetRipperGUI/Localizations/en_US.json

```diff
@@ -50,6 +51,10 @@
   "text_asset_format_text_description": "Export as a plain text file (.txt)",
   "text_asset_format_parse": "Parse",
   "text_asset_format_parse_description": "Export as a plain text file, but try to guess the correct file extension (e.g. JSON files get the .json extension)",
+  "shader_asset_format_dummy": "DummyShader",
+  "shader_asset_format_dummy_description": "Export the shader as a dummy-shader. While preserving data like \"Properties\" or \"FallBack\" it uses a general opaque shader code",
```

Comment:
Export the shader as a dummy shader. Although it preserves data like the Properties and FallBack, it uses general, opaque shader code.